media/public/include/MediaCommon.h

```diff
@@ -297,19 +300,23 @@ struct WebAudioShmInfo
  */
 struct WebAudioPcmConfig
 {
-    uint32_t rate;
-    uint32_t channels;
-    uint32_t sampleSize;
-    bool isBigEndian;
-    bool isSigned;
-    bool isFloat;
+    uint32_t rate;       /**< Rate of web audio (Hz) */
+    uint32_t channels;   /**< Number of channels */
+    uint32_t sampleSize; /**< Size of each sample (bytes or bits) */
```

Comment:
Can it really be in bits?

Reply:
Yeh, just asked Luke, and he checked and said its bits

Reply:
I'll remove bytes